.travis.yml

```diff
@@ -10,7 +10,7 @@ branches:
     - /^v\d+\.\d+(\.\d+)?(-\S*)?$/
     # … and release branches, e.g. jlc/v0.2.0-release
     - /^.+-release$/
-    - dynamic-services
+    - numeric-index-ids-for-cache
```

Comment:
The PR is CI-tested anyway, isn't it?

Reply:
The PR from the target branch is tested because it targets master; this one (as all the previous) does not as it targets  `numeric-index-ids-for-cache`.